ruby/.yardopts

```diff
@@ -0,0 +1,2 @@
+--no-private
```

Comment:
When does this file get used? and is `lib/doc` correct, or should it be `lib/stubs`?

Reply:
Good catch, I'll fix this.

It should be used when [https://www.rubydoc.info](https://www.rubydoc.info) indexes the gem. So I can use it to make sure it doesn't include e.g. the FFI namespace (which is what I meant to do with this but forgot).